### PR TITLE
Calculator component

### PR DESCRIPTION
**Pull request by julie-ify**
In this milestone, I; 

- [ ] Deleted all the boilerplate from CRA (text, images, styles).

- [ ] Created a directory called components.

- [ ] Inside components, I created a new calculator.js file.

- [ ] In Calculator.js, I created a Calculator component that matches this [design](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/images/calculator.png)
- [ ] Rendered the Calculator component from the App component

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  </div>
+);
+
+const Calculator = () => (
+  <div className="calc-container">
+    <div className="calc-wrap">
+      <Square />
+    </div>
+  </div>
+);
```
Comment (henatan99) [inline src/components/calculator.js:34]: - [ ] The Calculator.js component should be class-based as per the [project's requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md). It is the objective of this milestone to practice class-based React components. Kindly make changes here. :pray:

File: src/App.js [hunk 2]
```diff
@@ -1,29 +1,10 @@
...
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => (
+  <div className="container">
+    <Calculator />
+  </div>
+);
```
Comment (henatan99) [inline src/App.js:8]: - [ ] The App.js component should be class-based as per the project's requirements. It is the objective of this milestone to practice class-based React components. Kindly make changes here. :pray:

File: src/components/calculator.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    <span className="button">7</span>
+    <span className="button">8</span>
+    <span className="button">9</span>
+    <span className="button colored">&times;</span>
+    <span className="button">4</span>
+    <span className="button">5</span>
+    <span className="button">6</span>
+    <span className="button colored">&minus;</span>
+    <span className="button">1</span>
+    <span className="button">4</span>
```
Comment (henatan99) [inline src/components/calculator.js:19]: - [ ] There is a typo on this button. Kindly fix it. Despite the comments, your work is great. Your repo is very professional. 
image

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @julie-ify , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
### Hi @julie-ify  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the requested changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**